evm_loader/program/src/evm/opcode.rs

```diff
@@ -1196,8 +1201,7 @@ impl<B: Database> Machine<B> {
             return Ok(Action::Noop);
         }
 
-        let result = result.unwrap()?;
-        let return_data = Buffer::from_slice(&result);
+        let return_data = Arc::new(Buffer::from_byte_vec(result.unwrap()?));
```

Comment:
Can we return the error if result is None instead of using unwrap?

Reply:
@Deniskore thank you! It's impossible situation because of checking for `None` in line 1200. But I don't like `unwrap`'s  too, so I've rewritten this method to handle errors in more idiomatic way, take a look. :)